api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -424,6 +425,9 @@ message HttpConnectionManager {
   // Note that Envoy does not perform
   // `case normalization <https://tools.ietf.org/html/rfc3986#section-6.2.2.1>`
   google.protobuf.BoolValue normalize_path = 30;
+
+  // Used to configure the Local reply the type of the body
```

Comment:
This is not super clear. Can you write a more detailed description of what this field is doing?